sdk/dotnet/Waf/Policy.cs

```diff
@@ -308,18 +296,6 @@ public InputList<string> HttpListenerIds
         [Input("name")]
         public Input<string>? Name { get; set; }
 
-        [Input("pathBasedRuleIds")]
-        private InputList<string>? _pathBasedRuleIds;
-
-        /// <summary>
-        /// A list of URL Path Map Path Rule IDs from an `azure.network.ApplicationGateway`.
-        /// </summary>
-        public InputList<string> PathBasedRuleIds
-        {
-            get => _pathBasedRuleIds ?? (_pathBasedRuleIds = new InputList<string>());
-            set => _pathBasedRuleIds = value;
-        }
```

Comment:
Two properties removed in this file?

Reply:
Technically breaking but never actually worked it seems....

https://github.com/terraform-providers/terraform-provider-azurerm/pull/11196